com.jetbrains.IntelliJ-IDEA-Community.json

```diff
@@ -1,72 +1,77 @@
-          "tag": "0.19.1",
-          "url": "https://gitlab.gnome.org/GNOME/libsecret.git"
-        }
-      ]
-    }, {
-      "name": "idea",
-      "buildsystem": "simple",
-      "build-commands": [
-        "install --directory --mode=0755 /app/idea-IC/",
-        "tar --directory=/app/idea-IC/ --extract --file=ideaIC-2019.3.tar.gz --gunzip --strip-components=1",
-        "install -D --mode=0644 /app/idea-IC/bin/idea.svg /app/share/icons/hicolor/scalable/apps/com.jetbrains.IntelliJ-IDEA-Community.svg",
-        "install -D --mode=0755 entrypoint.sh /app/bin/idea",
-        "install -D --mode=0644 --target-directory=/app/share/applications/ com.jetbrains.IntelliJ-IDEA-Community.desktop",
-        "install -D --mode=0644 --target-directory=/app/share/metainfo/ com.jetbrains.IntelliJ-IDEA-Community.appdata.xml"
-      ],
-      "sources": [
+            "name": "libsecret",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dgtk_doc=false",
+                "-Dmanpage=false",
+                "-Dvapi=false"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "commit": "f576dfbe50f9aed22802ad0db0356d081f467d7a",
+                    "tag": "0.19.1",
+                    "url": "https://gitlab.gnome.org/GNOME/libsecret.git"
+                }
+            ]
+        },
         {
-          "type": "file",
-          "path": "entrypoint.sh"
-        }, {
-          "type": "file",
-          "sha256": "b2ebb30adffbde61cefe63598206b7b0a8e99bc4037130aca2c291c563fd4a60",
-          "url": "https://download.jetbrains.com/idea/ideaIC-2019.3.tar.gz",
-          "x-checker-data": {
-            "type": "jetbrains",
-            "code": "IIC"
-          }
-        }, {
-          "type": "file",
-          "path": "com.jetbrains.IntelliJ-IDEA-Community.appdata.xml"
-        }, {
-          "type": "file",
-          "path": "com.jetbrains.IntelliJ-IDEA-Community.desktop"
+            "name": "idea",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install --directory --mode=0755 /app/idea-IC/",
+                "tar --directory=/app/idea-IC/ --extract --file=ideaIC-2019.3.tar.gz --gunzip --strip-components=1",
+                "install -D --mode=0644 /app/idea-IC/bin/idea.svg /app/share/icons/hicolor/scalable/apps/com.jetbrains.IntelliJ-IDEA-Community.svg",
+                "install -D --mode=0755 entrypoint.sh /app/bin/idea",
+                "install -D --mode=0644 --target-directory=/app/share/applications/ com.jetbrains.IntelliJ-IDEA-Community.desktop",
+                "install -D --mode=0644 --target-directory=/app/share/metainfo/ com.jetbrains.IntelliJ-IDEA-Community.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "entrypoint.sh"
+                },
+                {
+                    "type": "file",
```

Comment:
Try with `"dest-filename": "ideaIC.tar.gz"` here to see if the filename can be predetermined and hence, used in `tar --directory=/app/idea-IC/ --extract --file=ideaIC.tar.gz --gunzip --strip-components=1` as such.